bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -80,10 +80,60 @@ spec:
                 exit 0
             }
             trap quit SIGTERM
+{{ if .EnableIPsec }}
```

Comment:
The OVS daemonsets will be going away; is there another place this could live?

Reply:
I guess that depends on where they are going and in what time frame. Do you have a suggestion? I could add something like this to ovn-kubernetes but because it needs to request signing from a controller that doesn't exist in the ovn kubernetes project, it didn't seem suitable to put it there. Will this go away in OCP 4.7 time frame?

Reply:
Just thinking about this. I will move this code into the ipsec container which can then also generate the keys (on the host filesystem) and configure ovs with them.

Reply:
Agree with @squeed The OVS daemonsets are just alive to help with logging since ovs is back to running on the hosts. It wouldn't make sense to put it in upstream... Why not just have a custom downstream container startup script for the IPsec Daemon that handles this? (Sorry if this is already what you meant @markdgray )

Reply:
Yes, as long as I can run `ovs-vsctl` in the ipsec pod, I can move all this there. This will simplify the design.

Reply:
@astoycos I didn't add a script to downstream yet. I don't think it makes sense to do that now as the feature is not enabled downstream. Maybe future versions.